File: dotnet/source/cmf.bus.berico/IOutboundEnvelopeProcessor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace cmf.bus.berico
{
    public interface IOutboundEnvelopeProcessor
    {
        void ProcessOutbound(ref Envelope envelope, ref IDictionary<string, object> context);
    }
}

File: java/src/main/java/cmf/bus/berico/IOutboundEnvelopeProcessor.java
package cmf.bus.berico;

import java.util.Map;

import cmf.bus.Envelope;
import cmf.bus.IDisposable;

public interface IOutboundEnvelopeProcessor extends IDisposable {

    void processOutbound(Envelope envelope, Map<String, Object> context);
}
